### DISCUSSION TITLE
Cursor and orderBy

### DISCUSSION THREAD

**Discussion by StanislavPavlov** [prisma/prisma#23631] [Q&A]
## Problem

Example: I have an array of the form [1, 2, 3, 4, 5]. I need to set the cursor to 2 and get the array in descending order, ie. [5, 4, 3].

**Comment by ludralph** [comment]
Hi @StanislavPavlov 👋 

Are you trying to paginate backwards using cursor based pagination? You can use the `findMany` method with the `cursor` and `orderBy` options. The cursor option allows you to set a bookmark in your result set, and the orderBy option allows you to sort the results in ascending or descending order. For example:
```js
const result = await prisma.yourModel.findMany({
  take: 3,
  cursor: { id: 2 },
  orderBy: { id: 'desc' },
});
```
Replace `yourModel` with the name of your model. This query will return the records with IDs 5, 4, and 3. See the [documentation](https://www.prisma.io/docs/orm/prisma-client/queries/pagination#sorting-and-cursor-based-pagination) for more information.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
